drivers/hmis/spec/models/hmis/client_alert_spec.rb

```diff
@@ -0,0 +1,26 @@
+###
```

Comment:
@gigxz Question about adding this spec file -- I was thinking to just write a quick test for the connections to `client` and `created_by` user, the things you recommended manually testing in the rails console, but I can't figure out around the
```
LoadError:
  cannot load such file -- /app/drivers/hmis/spec/models/client_alert_spec.rb
```
when trying to run these tests. Any ideas for what to try?
Also, should I create a FactoryBot Factory for the ClientAlerts at this point?

Reply:
That error looks to me like the test db needs to be migrated: `rails db:migrate RAILS_ENV=test`

We don't typically add "model" tests for simple classes like this.  Most likely instead we will have request tests for the Query / Mutation(s) that you'll add next (resolving alerts on a client; creating an alert; & deleting an alert).

It wouldn't hurt to add a factory in this PR if you want- You can also look at my PR for an example, we are doing similar things! https://github.com/greenriver/hmis-warehouse/pull/3923/files#diff-0c48dca452b35364c6e8e56c0a1d2b1553b586b4db59abe9c4de097ae80331a6

Reply:
Thank you! That was indeed the issue. Sounds good, I will add a factory here.